cmd/keeper/keeper.go

```diff
@@ -541,12 +542,22 @@ func (p *PostgresKeeper) Start() {
 	smTimerCh := time.NewTimer(0).C
 	updatePGStateTimerCh := time.NewTimer(0).C
 	publishCh := time.NewTimer(0).C
+	exitSignals := make(chan os.Signal, 1)
+	signal.Notify(exitSignals, syscall.SIGTERM, syscall.SIGINT, syscall.SIGQUIT)
+
 	for true {
 		select {
+		case sig := <-exitSignals:
+			log.Infof("Caught signal: %v.", sig)
+			log.Debugf("Stopping stolon keeper.")
```

Comment:
nit: Do you really need a separate level log message for this?
Also, `Debugf` implies format string with placeholders - `Debug` will do here.